### DISCUSSION TITLE
wire:dirty.attr: Adding disabled attribute to the button when the component is in "dirty" state

### DISCUSSION THREAD

**Discussion by 3yekn1** [livewire/livewire#6574] [3. Feature ideas]
I read the whole document and tried to use `wire:dirty` and `wire:dirty.remove` to fulfill the same requirement, but when I modify the data and then restore the data, the submit button disappears directly instead of disabled. as follow:

```html
<button
        type="submit"
        class="btn btnMore"
        wire:dirty.remove
        style='border: 0;'
        disabled
>Submit</button>

<button
        type="submit"
        class="btn btnMore"
        wire:dirty
>Submit</button>
```

It's really common and I'd like to get help

**Comment by wverhoogt** [comment]
Interestingly, that feature already exists, but is undocumented. If you add `wire:dirty.remove.attr="disabled"` to the first button, you´ll get the desired behaviour.

**Reply by 3yekn1** [reply]
Thank you very, very much for your help.

In livewire, the file uploads don't seem to be able to change the dirty state, do you have any good ideas on how to solve this problem?

**Comment by valpuia604** [comment]
> Interestingly, that feature already exists, but is undocumented. If you add `wire:dirty.remove.attr="disabled"` to the first button, you´ll get the desired behaviour.

Hi, I've tried with below code, class is working fine but I cannot get disabled attr, any idea
```html
<button class="lg-btn w-unset"
    type="submit"
    wire:dirty.class="primry-btn-2"
    wire:dirty.class.remove="primry-btn-2"
    wire:dirty.attr="disabled"
    wire:dirty.remove.attr="disabled"
    >
    Update Resume
</button>
```

**Reply by valpuia604** [reply]
Oh, I solved it like this..
```html
<button
    disabled
    wire:dirty.remove.attr="disabled" >
    Update Resume
</button>
```
